packages/validator/test/unit/utils/clock.test.ts

```diff
@@ -11,7 +11,8 @@ describe("util / Clock", function () {
 
   beforeEach(() => {
     controller = new AbortController();
-    vi.useFakeTimers();
+    // Feb 6, 2024 11:21:17.786 AM
+    vi.useFakeTimers({now: 1707214877786});
```

Comment:
what does this do and why is it needed to have a fixed value here?

Reply:
It's always better to have controlled env in unit tests and not rely on random values.

Reply:
But I don't get why we pass this specific date here, in other places we do

Date.now

https://github.com/ChainSafe/lodestar/blob/cb754f61164fdf9399d027de6b1ba12b33f24e7e/packages/beacon-node/test/unit/network/peers/datastore.test.ts#L13

or we pass 0

https://github.com/ChainSafe/lodestar/blob/cb754f61164fdf9399d027de6b1ba12b33f24e7e/packages/beacon-node/test/unit/network/processor/gossipQueues/indexed.test.ts#L27

Like would Date.now or 0 work here as well, and if not, why?

Reply:
I like that moment. 😄 

No specific reason, I was testing a specific case at some specific mili seconds.